src/redux/reducers/authReducers.js

```diff
@@ -1,107 +1,109 @@
 import { combineReducers } from 'redux';
 import {
   SIGNUP_REQUEST,
-  SIGNUP_SUCCESS,
   SIGNUP_FAILURE,
+  SIGNUP_SUCCESS,
   LOGIN_REQUEST,
-  LOGIN_SUCCESS,
   LOGIN_FAILURE,
+  LOGIN_SUCCESS,
   LOGOUT_REQUEST,
-  LOGOUT_SUCCESS,
   LOGOUT_FAILURE,
-  REQUEST_USER_INFO,
-  RECEIVE_USER_INFO_SUCCESS,
-  RECEIVE_USER_INFO_FAILURE
+  LOGOUT_SUCCESS,
+  GET_USER_INFO_REQUEST,
+  GET_USER_INFO_FAILURE,
+  GET_USER_INFO_SUCCESS
 } from '../actions/authActions';
 
-function authenticate(
-  state = {
-    isFetching: false,
-    user: [],
-    lastUpdated: null,
-    isAuthenticated: false,
-  },
-  action
-) {
+const initialState = {
+  lastUpdated: null,
+  isAuthenticated: false,
+  isFetching: false,
+  user: [],
+};
+
+function authenticate(state = initialState, action) {
```

Comment:
This is much cleaner!

Reply:
Thought it would be good to take it out considering our state tree will probably grow quite large.